notebooks/thunder_as_torch_compile_backend_tutorial.ipynb

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
<p>It would be nice if instead of having to write</p><p> </p>

<pre class="ql-syntax" spellcheck="false">backend = ThunderCompiler()
opt_foo1 = torch.compile(foo, backend=backend)
</pre><p> </p><p>We instead allowed practitioners to write</p><p> </p>

<pre class="ql-syntax" spellcheck="false">from thunder import thunderfx

tfoo = thunderfx(foo)
</pre><p> </p><p>Notebooks like this are a good opportunity to identify areas for improvement in our UX like this. We can create the thunderfx function so it accepts <code>**kwargs</code> to pass to the torch.compile call, too. </p><p> </p><p>It'd be great to see a PR changing this so we can update this doc. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Lightning-AI/lightning-thunder/pull/1524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>